extension-manifest-v3/src/background/tab-stats.js

```diff
@@ -55,14 +55,16 @@ async function updateTabStats(msg, sender) {
   const tabId = sender.tab.id;
   const stats = tabStats.get(tabId);
 
-  msg.urls
-    .filter((url) => !stats.trackers.some((t) => t.url === url))
-    .forEach((url) => {
-      const tracker = getTrackerFromUrl(url, stats.domain);
-      if (tracker) {
-        stats.trackers.push(tracker);
-      }
-    });
+  const newUrls = msg.urls.filter(
+    (url) => !stats.trackers.some((t) => t.url === url),
+  );
+
+  for (const url of newUrls) {
+    const pattern = await getMetadata(url, stats.sourceUrl);
```

Comment:
Would it make sense to run those requests in parallel?

Reply:
this is only for the startup